packages/toggle-crud-psr11-factories/src/RemoveStrategyFactory.php

```diff
@@ -12,9 +12,13 @@ final class RemoveStrategyFactory
 {
     public function __invoke(ContainerInterface $container): RemoveStrategy
     {
-        /** @var FeatureRepository $featureRepository */
-        $featureRepository = $container->get(FeatureRepository::class);
+        return self::create(
```

Comment:
Psalm doesn't like getting instances directly from the container

```suggestion
        /** @var FeatureRepository $featureRepository
        $featureRepository = $container->get(FeatureRepository::class);
        
        return self::create($featureRepository);
```

Reply:
> Psalm doesn't like getting instances directly from the container
> 
> 
> 
> ```suggestion
> 
>         /** @var FeatureRepository $featureRepository
> 
>         $featureRepository = $container->get(FeatureRepository::class);
> 
>         
> 
>         return self::create($featureRepository);
> 
> ```

I didn't know that, I'll setup my local env again to run Psalm